truss/truss_config.py

```diff
@@ -214,6 +214,28 @@ def to_list(self) -> List[Dict[str, str]]:
         return [item.to_dict() for item in self.items]
 
 
+@dataclass
+class BaseImage:
+    name: str = ""
+    python_executable_path: str = ""
+
+    @staticmethod
+    def from_dict(d):
+        name = d.get("name", "")
+        python_executable_path = d.get("python_executable_path", "")
```

Comment:
Can we add a validation step to ensure that all python_executable_path's are absolute paths (start with `/`) so that the symlinks actually work?